qiskit/visualization/circuit/circuit_utils.py

```diff
@@ -0,0 +1,637 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Common circuit visualization utilities."""
+
```

Comment:
Since this file is in a package called `circuit`, the `circuit_` prefix is perhaps a bit unnecessary.  As we're moving it, as well, we have a chance to make it (correctly) a private module.  Nothing other than our circuit drawers should be using it, so we can call it `_utils.py`.  That way, if we ever need to move it in the future, we won't need to have a deprecation warning.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/8306/commits/64844c9c120734626e9744569323ac521f64099a.